sensniff.py

```diff
@@ -45,23 +45,43 @@
 import logging
 import logging.handlers
 import struct
+import platform
+if platform.system() == 'Windows':
+	import win32pipe, win32file
+	import subprocess
+	
+	#open Wireshark
+	wireshark_cmd=['C:\Program Files\Wireshark\Wireshark.exe', r'-i\\.\pipe\wireshark','-k']
+	proc=subprocess.Popen(wireshark_cmd)
 #####################################
 ### Constants
 #####################################
-__version__ = '1.2'
+__version__ = '1.2.1'
 #####################################
 ### Default configuration values
 #####################################
-defaults = {
-    'device': '/dev/ttyUSB0',
-    'baud_rate': 460800,
-    'out_file': 'sensniff.hexdump',
-    'out_fifo': '/tmp/sensniff',
-    'out_pcap': 'sensniff.pcap',
-    'debug_level': 'WARN',
-    'log_level': 'INFO',
-    'log_file': 'sensniff.log',
-}
+if platform.system() == 'Windows':
+	defaults = {
+		'device': 'COM1',
+		'baud_rate': 460800,
+		'out_file': 'sensniff.hexdump',
+		'out_fifo': '/tmp/sensniff',
+		'out_pcap': 'sensniff.pcap',
+		'debug_level': 'WARN',
+		'log_level': 'INFO',
+		'log_file': 'sensniff.log',
+	}
+else:
+	defaults = {
+		'device': '/dev/ttyUSB0',
+		'baud_rate': 460800,
+		'out_file': 'sensniff.hexdump',
+		'out_fifo': '/tmp/sensniff',
+		'out_pcap': 'sensniff.pcap',
+		'debug_level': 'WARN',
+		'log_level': 'INFO',
+		'log_file': 'sensniff.log',
+	}
```

Comment:
Looks like the only change here is for 'device', so can we just use a separate variable and then keep a single defaults map? Would make the code more readable I think.